extensions/prometheus_metrics.py

```
@@ -0,0 +1,240 @@
+            asyncio.create_task(self._run_http_server(runner))
+        except ImportError:
+            logger.warning(
+                "aiohttp web module not available; Prometheus HTTP server not started. "
+                "Install aiohttp manually if you need the /metrics endpoint."
+            )
+
+    async def _run_http_server(self, runner: Any) -> None:
+        """Run the aiohttp server in the background."""
+        try:
+            await runner.setup()
+            site = web.TCPSite(runner, "0.0.0.0", 8000)
```

Flake8 (7.3.0)

[error] undefined name 'web'

(F821)